locale/zh-cn/docs/guides/dont-block-the-event-loop.md

```diff
@@ -118,7 +119,7 @@ app.get('/constant-time', (req, res) => {
 app.get('/countToN', (req, res) => {
   let n = req.query.n;
 
-  // n iterations before giving someone else a turn
+  // 先执行 n 次循环，才给其他请求执行机会
```

Comment:
I don't think it needs translating comments in the original codes because we want to keep it in origin. So please restore the rest part into the original English instead of Chinese here :)

Reply:
Updated.